runtime/vm/JVMImage.cpp

```diff
@@ -142,6 +261,34 @@ JVMImage::freeSubAllocatedMemory(void* address)
 	omrthread_monitor_exit(_jvmImageMonitor);
 }
 
+void
+JVMImage::registerEntryInTable(ImageTableHeader *table, UDATA entry)
```

Comment:
once you register a classloader, how can you ask for it back?

Reply:
I was wondering why do we need this (I saw the docs had classloader id passed in). Is this during warm run setting where we have to load back up. No one should access our heap. At least for write I only see looping through all the classLoaders at end to be needed.